_data/clients/weechat.yml

```
@@ -0,0 +1,5 @@
+name: Weechat
```

```suggestion
name: WeeChat
```

I wonder if it should say "WeeChat XMPP plugin" or something as WeeChat doesn't support XMPP out of the box.

---

Like so?